src/TorchSharp/Tensor/Tensor.Factories.cs

```diff
@@ -950,6 +950,22 @@ public static Tensor normal(Tensor mean, Tensor std, torch.Generator? generator
             return randn(mean.shape, generator: generator, device: std.device) * std + mean;
         }
 
+        /// <summary>
+        /// Similar to the function above, but the means and standard deviations are shared among all drawn elements. The resulting tensor has size given by size.
+        /// </summary>
+        /// <param name="mean">The mean for all distributions</param>
+        /// <param name="std">The standard deviation for all distributions</param>
+        /// <param name="size">A sequence of integers defining the shape of the output tensor.</param>
+        /// <param name="dtype"></param>
+        /// <param name="device"></param>
+        /// <param name="requires_grad"></param>
+        /// <param name="generator">An optional random number generator</param>
+        /// <returns></returns>
+        public static Tensor normal(double mean, double std, ReadOnlySpan<long> size, torch.ScalarType? dtype = null, torch.Device? device = null, bool requires_grad = false, torch.Generator? generator = null)
+        {
+            return randn(size, dtype: dtype, device: device, requiresGrad: requires_grad, generator: generator) * std + mean;
```

Comment:
If device is 'null,' should it use the std.device instead of the default device, perhaps?

Reply:
`torch.rand()` should handle the case where `device` is `null`. As the user may call it like `torch.rand(size, dtype: torch.float64)` without `device`. `torch.InitializeDevice(null)` falls back to `torch.CPU`. We may want to call C++ `torch::rand()` function without device option if we want to make sure it is compatible with PyTorch?

Reply:
@kaiidams -- I was confused. I was thinking that std and mean were tensors, and that the device should follow them unless non-null. std and mean are, of course, doubles, so the default device makes sense.